protobuf/flow/access/access.proto

```diff
@@ -424,6 +451,55 @@ message GetAccountAtBlockHeightRequest {
   uint64 block_height = 2;
 }
 
+message GetAccountBalanceAtLatestBlockRequest {
+  bytes address = 1;
+}
+
+message GetAccountBalanceAtBlockHeightRequest {
+  bytes address = 1;
+  uint64 block_height = 2;
+}
+
+message AccountBalanceResponse {
+  uint64 balance = 1;
+  uint64 available_balance = 2;
```

Comment:
as noted in the flow-go PR, let's remove `available_balance` for now. we can add it back later.